src/main/java/net/pms/util/OpenSubtitle.java

```diff
@@ -515,18 +550,239 @@ public static String getHash(File file) throws IOException {
 				episodeName = "";
 			}
 
+			String movieOrShowName = StringEscapeUtils.unescapeHtml4(name);
+
 			return new String[]{
-				ImdbUtil.ensureTT(m.group(1).trim()),
+				imdbId,
 				episodeName,
-				StringEscapeUtils.unescapeHtml4(name),
-				m.group(3).trim(), // Season number
-				m.group(4).trim(), // Episode number
-				m.group(5).trim()  // Year
+				movieOrShowName,
+				m.group(4).trim(), // Season number
+				m.group(5).trim(), // Episode number
+				m.group(3).trim()  // Year
 			};
 		}
 		return null;
 	}
 
+	public static void backgroundLookupAndAdd(final File file, final DLNAMediaInfo media) {
+		if (!PMS.get().getDatabase().isOpenSubtitlesMetadataExists(file.getAbsolutePath(), file.lastModified())) {
+			final boolean overTheTopLogging = false;
+			String[] metadataFromFilename = FileUtil.getFileNameMetadata(file.getName());
```

Comment:
@SubJunk this part of the code should be added to `registerExternalSubtitles` in the PR  #1372 but I don't have any idea how to do it.